test/refract-test.coffee

```diff
@@ -408,5 +408,19 @@ describe('Transformations • Refract', ->
         assert.strictEqual(applicationAst.sections[0].resources[0].requests.length, 1)
       )
     )
+
+    describe('‘x-summary’ and ‘x-description’', ->
+      applicationAst = null
+
+      before(->
+        applicationAst = convertToApplicationAst(
+          require('./fixtures/refract-parse-result-x-summary.json')
+        )
+      )
+
+      it('Has the correct number of resources', ->
```

Comment:
Can we also test the `x-summary` and `x-description` content is present and matches what we expect here?